recipe/meta.yaml

```diff
@@ -34,8 +34,6 @@ requirements:
     - libsodium
   run:
     - python
-    - zeromq
-    - libsodium
```

Comment:
Removing zeromq and libsodium from the run section because they are already host dependencies and brought as run exports.